### ISSUE TITLE
Use tilemap to store the stages

### ISSUE THREAD

**Issue by private-yusuke** [ckkpd/breakout-like#4]
**Is your feature request related to a problem? Please describe.(何のための機能ですか)**
Setting up a scene for each stage makes the developers harder to make this game so we use tilemap to make the development easier and simpler.

**Describe the solution you'd like(どんな機能ですか)**
It prevents us to make a lot of scenes for each stages. Setting up stages using tilemap is much more simpler compared to current way.
Basically people use tilemap for 1x1 cells, but don't worry since it supports n x m where n and m is integer.

**Describe alternatives you've considered(それ以外にどんなことを考えましたか)**
+ write a stage generator in C# for each stages
  - doesn't make sense since it is not using Unity's great feature.

**Additional context(備考)**
We might use ScriptableObject to store stages.

**Comment by private-yusuke** [comment]
I won't implement this feature because I couldn't deal with Unity's Tilemap as I expected.
